lib/WebService/MinFraud/Record/BillingAddress.pm

```diff
@@ -49,7 +49,7 @@ country as that of the IP address.
 =head2 is_postal_in_city
 
 Returns a boolean indicating whether the billing postal code is in the
-billing city.
+billing city, or an undef if the address is not in USA.
```

Comment:
I think this one is probably phrased as something like "The attribute will not be set if the address is not in the USA." We provide predicate methods so the user can check whether it has been set.